deployments/roles/config_openresty/templates/internal.nginx

```diff
@@ -0,0 +1,141 @@
+upstream pss {
+    server localhost:{{ port_pss }};
+}
+
+upstream grafana {
+    server localhost:{{ port_grafana }};
+}
+
+upstream prometheus {
+    server localhost:{{ port_prometheus }};
+}
+
+upstream openapi {
+    server localhost:{{ port_openapi }};
+}
+
+server {
+    listen              80;
+    resolver            8.8.8.8;
+    server_name         {{ hostname }};
+    return 301          https://$host$request_uri;
+}
+
+server {
+    listen              443 ssl;
+    resolver            8.8.8.8;
+    server_name         {{ hostname }};
+
+    # ssl config
+    ssl_certificate     /etc/openresty/ssl/{{ hostname }}/cert.pem;
+    ssl_certificate_key /etc/openresty/ssl/{{ hostname }}/key.pem;
+    ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
+    ssl_ciphers         HIGH:!aNULL:!MD5;
+
+    # log config
+    access_log          {{ dir_openresty_internal_log }}/access.log;
+    error_log           {{ dir_openresty_internal_log }}/error.log;
+
+    # default redirection
+    location / {
+        return 301 /grafana;
+    }
+
+    location ^~ /.well-known/acme-challenge {
+        root /etc/openresty/ssl/$host;
+    }
+
+    location /sqlsmith/results {
+        alias /var/www/sqlsmith/results;
+        autoindex on;
+    }
+
+    location /sqlsmith/logs {
+        alias /var/www/sqlsmith/logs;
+        autoindex on;
+        types {
+            text/plain log csv;
+        }
+    }
+
+    location /sqlancer/logs {
+        alias /var/www/sqlancer/logs;
+        autoindex on;
+        types {
+            text/plain log csv;
+        }
+    }
```

Comment:
We may want to move these logs somewhere else. Like add them as a file to the repo or something because I doubt anyone will remember they are on the server once we are gone.

Reply:
you mean sqlancer logs? or the proxy config?

Reply:
I mean the actual logs. But if we move the logs then we can probably remove the proxy config.

Reply:
I think we can ask Andy if he still wants the sqlancer logs to be somewhere. If yes, I will change the proxy target dir accordingly; otherwise, I just remove it.